niedoida/include/niedoida/niedoida.hpp

```diff
@@ -122,7 +121,11 @@ namespace niedoida {
         double integral_threshold;
     };
 
-
+    struct StateSymmetryRepresentation {
+        static arma::uvec degeneracy;
+        static arma::uvec cc_sizes;
+        static arma::uvec mo_symmetry;
+    };
 
```

Comment:
Ugh. Please get rid of the static field and NEVER use them again unless you really know what you are doing. Actually, just get rid of the whole thing altogether, after all you don't really use it anyway.